docs/upgrades/migrate_docker2containerd.md

```diff
@@ -65,6 +65,13 @@ apt-get install pigz
 
 ### 5) Run `cluster.yml` playbook with `--limit`
 
+> NOTE: If your cluster is previously setup using `etcd_kubeadm_enabled: true` then you will need to update the static manifest `/etc/kubernetes/manifests/etcd.yaml` on all etcd nodes with the new version before this step.
+>
+> ```diff
+> -     image: quay.io/coreos/etcd:v3.4.13
+> +     image: quay.io/coreos/etcd:v3.5.0
```

Comment:
Strictly speaking this is not related to the docker to containerd migration but kube version upgrade.

Reply:
Yes, I just verified this. So upgrading first and then migrate according to the documentation works great. 
Seems like I was just a little bit to greedy and tried to upgrade from 2.17.1 -> 2.18.0 at the same time as this migration and then this doesn't work.
You can go ahead and close this one